packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -386,6 +386,44 @@ class ShortcutManager extends ChangeNotifier with Diagnosticable {
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+/// ```dart imports
+/// import 'package:flutter/services.dart';
+/// ```
+///
+/// ```dart preamble
+/// class Increment extends Intent {}
+///
+/// class Decrement extends Intent {}
+/// ```
+///
+/// ```dart
+/// int count = 0;
+///
+/// Widget build(BuildContext context) {
+///   return Shortcuts(
+///     shortcuts: <LogicalKeySet, Intent> {
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyI): Increment(),
+///       LogicalKeySet(LogicalKeyboardKey.alt, LogicalKeyboardKey.keyD): Decrement(),
+///     },
+///     child: Actions(
+///       actions: <Type, Action<Intent>> {
+///         Increment: CallbackAction<Increment>(
+///          onInvoke: (Increment intent) => setState(()=> count = count + 1),
+///         ),
+///         Decrement: CallbackAction<Decrement>(
+///          onInvoke: (Decrement intent) => setState(()=> count = count - 1),
+///         ),
+///       },
+///       child: Center(
+///         child: Text('$count'),
+///       ),
```

Comment:
You will need to wrap the text widget in a `Focus`, since the shortcuts only work when there's a keyboard-focusable widget. So, something like:
```dart
Focus(
  autofocus: true,
  child: Text('count: $count'),
),
```